tests/bc/run.sh

```diff
@@ -20,9 +20,10 @@ do
 done
 
 #put OS and Device type here
-SUFFIX="ubuntu12.04_k40c"
-EXCUTION=$exe_file
-DATADIR="/data/gunrock_dataset/large"
+SUFFIX="ubuntu16.04_TitanV"
+EXCUTION="./bin/test_bc_9.1_x86_64" #$exe_file
```

Comment:
Typo, `EXCUTION` => `EXECUTION`

Reply:
Also needs to be fixed in some lines where its being used.

Reply:
fixed